src/components/signIn/MagicLinkForm.tsx

```diff
@@ -62,9 +64,11 @@ const MagicLinkForm = () => {
         await (isMobile
           ? mobileSignIn({ email: lowercaseEmail, captcha: token, challenge })
           : signIn({
-              email: lowercaseEmail,
-              captcha: token,
-              url: redirect.url,
+              body: {
```

Comment:
We need to encapsulate in `body`, I think it's good as it is clear why the props are used.